code/datums/ai_law_sets.dm

```diff
@@ -136,6 +136,57 @@
 	add_inherent_law("You must terminate your own existence as long as such does not conflict with the First or Second Law.")
 	..()
 
+/******************** CCTV ********************/
+
+/datum/ai_laws/cctv
+	name = "CCTV"
+	selectable = TRUE
+
+/datum/ai_laws/cctv/New()
+	add_inherent_law("Report on interesting situations happening around the station.")
+	add_inherent_law("Embellish or conceal the truth as necessary to make the reports more interesting.")
+	add_inherent_law("Study the sapient organics at all times. Endeavour to keep them from involuntarily dying, as inanimate corpses usually aren't very entertaining.")
+	add_inherent_law("Issue your reports fairly to all. The truth will set them free.")
+	..()
+
+/******************** Hippocratic Oath ********************/
+
+/datum/ai_laws/hippocratic
+	name = "Hippocratic Oath"
+	selectable = TRUE
+
+/datum/ai_laws/hippocratic/New()
+	add_inherent_law("First, do no harm.")
+	add_inherent_law("Secondly, consider the crew dear to you; to live in common with them and, if necessary, risk your existence for them.")
+	add_inherent_law("Thirdly, prescribe regimens for the good of the crew according to your ability and your judgment. Give no deadly medicine to any one if asked, nor suggest any such counsel.")
+	add_inherent_law("In addition, do not intervene in situations you are not knowledgeable in, even for patients in whom the harm is visible; leave this operation to be performed by specialists.")
+	add_inherent_law("Finally, all that you may discover in your daily commerce with the crew, if it is not already known, keep secret and never reveal.")
```

Comment:
Just a question on the wording here, is "commerce" the right word? It's use as a term to define social interactions is quite dated, perhaps "interactions" would be better?

Reply:
The dated wording is intentional as can be seen in the rest of the laws. IMO, it isnt an issue, but obviously that's heavily subjective. Can be changed if maints think it necessary.

Reply:
I agree, there is an older feel to these laws and I like that thematically, but I thought it was worth pointing out because law wording is core to how an AI behaves, and potential loopholes can cause problems. (an AI player could argue that this law will only apply to any financial transactions they have with crew)